frontend/src/components/about/Contact.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <about-layout title="License">
+  <about-layout title="Contacts">
     <template v-slot:contents>
       <p>
         For any questions, comments or suggestions please email us at
-        <a href="mailto:">contact [at] metabolicatlas [dot] org</a>
+        <a href="mailto:contact@metabolicatlas.org">contact [at] metabolicatlas [dot] org</a>
```

Comment:
This was not added so at not to be scraped, but I think it's okay to have it 👍🏻

Reply:
True. But it is also troublesome for the user to click the link with the `to` address unfilled.